src/features/DeFiZap/components/ZapCard.tsx

```diff
@@ -139,14 +149,15 @@ const ZapCardRiskProfile = styled('div')`
   margin-left: 0.5em;
 `;
 
-const ZapCardButton = styled(Button)<{ width: string }>`
+const ZapCardButton = styled(Button)`
   @media (min-width: ${BREAK_POINTS.SCREEN_XS}) and (max-width: ${BREAK_POINTS.SCREEN_MD}) {
     width: 100%;
   }
   display: flex;
+  flex: 1 auto;
   justify-content: center;
   align-items: center;
-  width: ${(props) => props.width};
+  width: 100%;
```

Comment:
Copy don’t respect designs, `add` button should be smaller than `withdraw` button

Reply:
Yeah, I realize, but in the task description, it says, they should be 50%/50%. https://app.clubhouse.io/mycrypto/story/5978/defizap-design-improvements-and-rebranding-zapper

Reply:
You’re right ! All good then :)